src/platform/packages/shared/response-ops/rule_form/src/rule_details/rule_dashboards.test.tsx

```diff
@@ -214,7 +214,7 @@ describe('RuleDashboards', () => {
       expect(searchInput).toHaveValue('Dashboard 1');
 
       expect(mockDashboardServiceProvider().fetchDashboards).toHaveBeenCalledWith(
-        expect.objectContaining({ limit: 100, text: 'Dashboard 1*' })
+        expect.objectContaining({ query: { limit: 100, text: 'Dashboard 1*' }, spaces: ['*'] })
```

Comment:
Could you please add a test with an active space different from the default one (`*`)?

Reply:
I have already added test for this https://github.com/elastic/kibana/pull/225467/files/2d25c213d424142b71d63dd44c5d02baf02dda34